data_steward/tools/snapshot_serology_dataset.py

```diff
@@ -0,0 +1,202 @@
+# coding=utf-8
+"""
+Script to generate the serology dataset in the form C{release_tag}_antibody_quest
+containing tables from R2020q4r1_antibody_quest as done in DC-1981.
+These will be used to create C{release_tag}_serology in the output project.
+Issue: DC-2263
+"""
+# Python imports
+import argparse
+import logging
+
+# Third party imports
+from google.cloud.bigquery import Dataset
+
+# Project imports
+from common import CDR_SCOPES, JINJA_ENV, PERSON
+from utils import auth, pipeline_logging
+from gcloud.bq import BigQueryClient
+
+LOGGER = logging.getLogger(__name__)
+
+ISSUE_NUMBER = 'DC2263'
+
+S_TITER = 'titer'
+S_ROCHE_ORTHO = 'roche_ortho'
+S_TEST = 'test'
+S_RESULT = 'result'
+
+SEROLOGY_TABLES = [PERSON, S_TITER, S_ROCHE_ORTHO, S_TEST, S_RESULT]
+
+PERSON_QUERY = JINJA_ENV.from_string("""
+CREATE TABLE `{{project_id}}.{{dest_dataset_id}}.serology_person`
+LIKE `{{project_id}}.{{source_dataset_id}}.person` AS
```

Comment:
@nishanthpp93 for these tables aren't we adding the schemas and using them to create the tables? is there any benefit in doing it like this?